scripts/start_integration_test_contracts_with_docker.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Get the latest Finschia tag
+TAG=$(curl "https://api.github.com/repos/Finschia/finschia/tags" | jq -r '.[0].name')
+
+# Copy init_single.sh from Finschia/finschia/init_single.sh
+init_single=$(mktemp)
+curl "https://raw.githubusercontent.com/Finschia/finschia/${TAG}/init_single.sh" -o $init_single
+
+# Remove v from tag prefix
+# ex. v1.0.0 -> 1.0.0
+TAG=$(echo "$TAG" | cut -c 2-)
+TEST_DOCKER_IMAGE=finschia/finschianode:${TAG}
+docker pull finschia/finschianode:${TAG}
+
+export FNSAD="docker run -i --rm -p 26656:26656 -p 26657:26657 -v ${HOME}/.finschia:/root/.finschia ${TEST_DOCKER_IMAGE} fnsad"
+
+# run Finschia/finschia/init_single.sh
+bash $init_single
+
+container_id=$(docker run -d -p 26656:26656 -p 26657:26657 -v ${HOME}/.finschia:/root/.finschia ${TEST_DOCKER_IMAGE} fnsad start)
+
+repository_top=$(git rev-parse --show-toplevel)
```

Comment:
This works well if only this is executed in the repository. How about getting this script directory with `basedir ${0}` and using it?

Reply:
fixed here.
[fix: change to use dirname to get absolute path](https://github.com/Finschia/finschia-wasm/pull/16/commits/ade00e7b26db3929509b885f60a42c9500222fe2)